lco_ingester/settings/settings.py

```diff
@@ -39,3 +39,6 @@ def get_tuple_from_environment(variable_name, default):
 
 # Processed files RabbitMQ Exchange Name
 PROCESSED_EXCHANGE_NAME = os.getenv('PROCESSED_EXCHANGE_NAME', 'archived_fits')
+
+# Optionally submit files to fits queue
+POSTPROCESS_FILES = os.getenv('POSTPROCESS_FILES', True)
```

Comment:
Buggy. You need `ast.literal_eval(os.getenv('POSTPROCESS_FILES', 'True'))`.

`os.getenv()` only returns strings. It doesn't automatically convert to bool. :-)

Reply:
you're right, thanks